libs/ledger-live-common/src/account/serialization.ts

```diff
@@ -818,26 +827,37 @@ export function fromAccountRaw(rawAccount: AccountRaw): Account {
             fromPolkadotResourcesRaw(polkadotResourcesRaw);
       }
       break;
-    case "elrond":
-      (res as ElrondAccount).elrondResources = fromElrondResourcesRaw(
-        (rawAccount as ElrondAccountRaw).elrondResources
-      );
+    case "elrond": {
+      const elrondResourcesRaw = (rawAccount as ElrondAccountRaw)
+        .elrondResources;
+
+      if (elrondResourcesRaw)
+        (res as ElrondAccount).elrondResources =
+          fromElrondResourcesRaw(elrondResourcesRaw);
       break;
-    case "cardano":
-      (res as CardanoAccount).cardanoResources = fromCardanoResourceRaw(
-        (rawAccount as CardanoAccountRaw).cardanoResources
-      );
+    }
+    case "cardano": {
+      const cardanoResourcesRaw = (rawAccount as CardanoAccountRaw)
+        .cardanoResources;
+      (res as CardanoAccount).cardanoResources =
+        fromCardanoResourceRaw(cardanoResourcesRaw);
       break;
-    case "solana":
-      (res as SolanaAccount).solanaResources = fromSolanaResourcesRaw(
-        (rawAccount as SolanaAccountRaw).solanaResources
-      );
+    }
+    case "solana": {
+      const solanaResourcesRaw = (rawAccount as SolanaAccountRaw)
+        .solanaResources;
+      (res as SolanaAccount).solanaResources =
+        fromSolanaResourcesRaw(solanaResourcesRaw);
       break;
-    case "crypto_org":
+    }
+    case "crypto_org": {
+      const cryptoOrgResourcesRaw = (rawAccount as CryptoOrgAccountRaw)
+        .cryptoOrgResources;
       (res as CryptoOrgAccount).cryptoOrgResources = fromCryptoOrgResourcesRaw(
-        (rawAccount as CryptoOrgAccountRaw).cryptoOrgResources
+        cryptoOrgResourcesRaw
       );
       break;
+    }
```

Comment:
Why not check `if (xxxRawResources)` for Bitcoin, Cardano, Solana and CryptoOrg ? Same as you did for the others.

Reply:
oops will do thx i forgot about this one